Source/abci/Importer/aiMeshSchema.h

```diff
@@ -334,6 +494,64 @@ void aiMeshSchema<T, U>::updateSummary()
         }
     }
 
+    for (size_t i = 0; i < m_attributes_param.size(); ++i) {
+
+        auto attrib = m_attributes_param[i];
+
+        switch (attrib->type1) {
+        case(aiPropertyType::Unknown):
+        {
+            auto& param = *static_cast<AbcGeom::IV2fGeomParam*>(attrib->data);
+
+            if (param.valid() && param.getNumSamples() > 0 && param.getScope() != AbcGeom::kUnknownScope)
+            {
+                summary.has_attributes_prop.push_back(true);
+                summary.has_attributes++;
```

Comment:
[ABC-486](https://jira.unity3d.com/browse/ABC-486) : This will contain the number of custom attributes.  We will pass this value to the C# before allocating the memory to the attributes.